sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -223,6 +224,17 @@ def from_message_content(  # pylint: disable=unused-argument
         event_data.content_type = content_type
         return event_data
 
+    @classmethod
+    def from_bytes(cls, message: bytes) -> "EventData":
+
+        amqp_message = decode_payload(memoryview(message))
+        print(amqp_message)
+
+        event_data = cls(body="")
```

Comment:
just curious why does body get initialized as ""? otw looks fine to me :)

Reply:
we should add a description of what sort of bytes we expect here.